examples/const-and-final/const_and_final.description

```diff
@@ -1,3 +1,3 @@
-// `const` means immutable and known at compile-time. Its type is singleton: set containing single value.
-// A `variable` or a `class` field can be declared as `final`, meaning it cannot be assigned to, after
+// `const` means immutable and known at compile-time. Its type is singleton: a set containing a single value.
+// A `variable` or a `class` field can be declared as `final`, which means it cannot be assigned after
```

Comment:
```suggestion
// A variable or a class field can be declared as `final`, which means it cannot be assigned after
```
Since,
- "variable" is not a keyword
- "class" is not used in its keyword context